src/Sentry/iOS/SentrySdk.cs

```diff
@@ -0,0 +1,23 @@
+using Sentry.iOS;
+
+namespace Sentry;
+
+public static partial class SentrySdk
+{
+    static partial void InitNative(SentryOptions options)
+    {
+        var o = new Sentry.iOS.SentryOptions();
+        o.Dsn = options.Dsn;
+        o.Debug = options.Debug;
+        SentrySDK.StartWithOptionsObject(o);
+
+        // Blocked not getting executed:
+        // SentrySDK.StartWithConfigureOptions(o =>
+        // {
+        //     o.Dsn = options.Dsn;
+        //     o.Debug = options.Debug;
+        // });
+
+        SentrySDK.CaptureMessage("Testing binding to iOS");
```

Comment:
Works:

![image](https://user-images.githubusercontent.com/1633368/174449006-ed1e73b9-956f-43e3-9c79-cce9a8eabfc6.png)


Stack trace looks like that because we don't upload symbols yet on build